tests/suite-unit/PluginFlyvemdmEntityConfig.php

```diff
@@ -116,7 +116,7 @@ public function providerPrepareInputForUpdate() {
             [],
             [
                'You are not allowed to change the device limit',
-               'You are not allowed to download URL of the MDM agent',
+               'You are not allowed to change the download URL of the MDM agent',
```

Comment:
This should go in the PR I did today.

Reply:
That PR is merged?

Reply:
I guess you fixed a unit test. If so, i t seems my PR did not passed tests but was merged anyway. I'll check this morning

Reply:
My PR was merged despite unit tests failed. This line should fix the failed test.